Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/AsyncPDFRenderer.mm

```diff
@@ -181,11 +181,11 @@
     }
 }
 
-void AsyncPDFRenderer::transferBufferToMainThread(RefPtr<ImageBuffer>&& imageBuffer, const TileForGrid& tileInfo, const FloatRect& tileRect)
+void AsyncPDFRenderer::transferBufferToMainThread(RefPtr<ImageBuffer>&& buffer, const TileForGrid& tileInfo, const FloatRect& tileRect)
```

Comment:
y tho

Reply:
Chaotically trying to fix the bizarro build error, don't understand it and have never seen it locally